src/main/java/org/folio/circulation/domain/LastCheckIn.java

```diff
@@ -0,0 +1,94 @@
+
+package org.folio.circulation.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.vertx.core.json.JsonObject;
+import org.joda.time.DateTime;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.invoke.MethodHandles;
+import java.util.UUID;
+
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static org.folio.circulation.support.JsonPropertyWriter.write;
+
+/**
+ * Information about when an item was last scanned in the Inventory app.
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "dateTime", "servicePointId", "staffMemberId" })
+public class LastCheckIn {
```

Comment:
Similar to my comments on the inventory pull request (I'm assuming this is a copy and paste of the generated class from storage). 

I think this class can be simplified to 3 immutable fields and a `toJson` method that writes them out explicitly.

There might also need to be properties, if this needs mapping into responses as well.

Reply:
Done.